### DISCUSSION TITLE
progessive speed

### DISCUSSION THREAD

**Discussion by deeererr** [mdolab/dafoam#340] [General]
Hello, do you know if it is possible to have a progressive speed inlet insted of fixed values?

**Comment by friedenhe** [comment]
Do you want to change Uin within a simulation? Or do you want to set different Uin for each optimization iteration? The former is not supported because it will eventually become an unsteady simulation. The latter is possible by using a similar strategy to the angle of attack function in the NACA0012 case.

**Reply by deeererr** [reply]
Hello, I would like to introduce a progressive speed (for example a linear increase during the iterations of calculation), but keep the same speed for each optimization. I want to do this because I have problems of divergences from the initialization, and I think that putting a progressive speed could help the convergence of calculations for the initialization. So if I understand correctly, this is not possible to do this?

**Reply by friedenhe** [reply]
If OpenFOAM has this BC, you can add it to DAFoam. Also, I doubt that adding this BC will help the convergence.
